src/components/Url.ts

```diff
@@ -0,0 +1,123 @@
+import {
```

Comment:
Maybe move this file out of `/components` since it doesn't contain a React component?

Reply:
Actually I'd suggest merging it with `ShareLink.ts` since they seem quite closely related

Reply:
Great suggestion, I've combined the two